### ISSUE TITLE
`nx-cloud start-ci-run` failed with yarn 2

### ISSUE THREAD

**Issue by serko-yang** [nrwl/nx#13612]
### Current Behavior

When run command `nx-cloud start-ci-run` it fails with error:

```bash
Error: @nrwl/nx-cloud tried to access open, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
```

It has to install `open` package to root to fix it.

### Expected Behavior

It should run the command without adding `open` package

### Steps to Reproduce

1. Install `nx-cloud` at the workspace as instructed
2. Run `nx-cloud start-ci-run`
3. Notice the command fails

### Nx Report

```shell
Node : 16.17.1
   OS   : darwin arm64
   yarn : 3.2.3
   
   nx : 15.2.4
   @nrwl/angular : Not Found
   @nrwl/cypress : 15.2.4
   @nrwl/detox : Not Found
   @nrwl/devkit : Not Found
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : 15.2.4
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 15.2.4
   @nrwl/js : 15.2.4
   @nrwl/linter : 15.2.4
   @nrwl/nest : Not Found
   @nrwl/next : 15.2.4
   @nrwl/node : Not Found
   @nrwl/nx-cloud : 15.0.2
   @nrwl/nx-plugin : Not Found
   @nrwl/react : 15.2.4
   @nrwl/react-native : Not Found
   @nrwl/rollup : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 15.2.4
   @nrwl/web : 15.2.4
   @nrwl/webpack : Not Found
   @nrwl/workspace : 15.2.4
   typescript : 4.8.2
```

**Comment by StalkAltan** [comment]
@serko-yang Thanks for reporting this and apologies for the delay, I will be taking a look in the upcoming week.

**Comment by StalkAltan** [comment]
This has been released in Nx Cloud 16.0.2.
